automx2/generators/outlook.py

```diff
@@ -29,6 +29,8 @@
 from automx2.ldap import STATUS_SUCCESS
 from automx2.model import Domain
 from automx2.model import Server
+from automx2.util import expand_placeholders
```

Comment:
Remove this.

Reply:
Can I keep it?

Reply:
Yes, because you are now using this. You were importing `_sanitise` and this was not needed then.